sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Style nit - you could use backticks / code font for <code>ml_domain = 'MACHINE_LEARNING'</code>  and <code>ml_task = 'CLASSIFICATION' </code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>